controllers/manager.js

```diff
@@ -1,7 +1,41 @@
+const db = require('../db');
+const PS = require('pg-promise').PreparedStatement;
 const { Roles } = require('../auth/index');
 const { getNextUid } = require('../controllers/user');
-const { addRider, addStaff } = require('../db/fillTableMethods');
+const { addRider, addStaff, addRestaurant, addGlobalPromotion } = require('../db/fillTableMethods');
```

Comment:
Don't user fillTableMethods, they are supposed to be used for filling db with dummy data and not using PreparedStatements.